### DISCUSSION TITLE
AlphaTab helps students learn guitar

### DISCUSSION THREAD

**Discussion by danmazz2001** [CoderLine/alphaTab#591] [Show and tell]
I am the founder of [Moosiko](moosiko.com), and online guitar & ukulele practice platform for schools in the US. AlphaTab helps students learn melodies, riffs, and solos in our practice platform. We evaluated a number of tools to display TAB/staff but the customization and flexibility of AlphaTab makes it a great solutions for us.

**Comment by Danielku15** [comment]
Looks like a nice project. Mind sharing some screenshots, videos or free sample page with with us how alphaTab looks like in your platform? Unfortunately with subscribing to your service we cannot check out how it looks like. 😉
